spec/models/edition_spec.rb

```diff
@@ -27,12 +27,13 @@
 
   describe ".create_initial" do
     let(:document) { build(:document) }
+    let(:document_type_id) { build(:document_type, path_prefix: "/prefix").id }
     let(:user) { build(:user) }
 
     it "creates a current edition" do
       edition = Edition.create_initial(
         document: document,
-        document_type_id: document.document_type_id,
+        document_type_id: document_type_id,
```

Comment:
This last commit should be fixed up with the first ones too.

Reply:
👍